src/libstd/rt/unwind/gcc.rs

```diff
@@ -98,6 +99,20 @@ pub mod eabi {
                                 context: *mut uw::_Unwind_Context)
             -> uw::_Unwind_Reason_Code;
     }
+    #[cfg(target_os = "nacl")]
+    unsafe fn __gcc_personality_v0(_version: c_int,
+                                   _actions: uw::_Unwind_Action,
+                                   _exception_class: uw::_Unwind_Exception_Class,
+                                   _ue_header: *mut uw::_Unwind_Exception,
+                                   _context: *mut uw::_Unwind_Context) ->
+        uw::_Unwind_Reason_Code
+    {
+        // we just need to be here to prevent linker errors.
+        // based on my analysis of PNaClSJLJ.cpp and ExceptionInfoWriter.cpp,
+        // the personality functions are just discarded without any consultation.
+        use core::intrinsics::abort;
+        abort()
+    }
```

Comment:
Does nacl just not have this function? It may want an entirely separate code path as this will basically translate to abort-on-panic otherwise.